dashboard/test/models/peer_review_test.rb

```diff
@@ -475,6 +475,31 @@ class PeerReviewTest < ActiveSupport::TestCase
     peer_review.update!(audit_trail: 'unrelated change')
   end
 
+  test 'create_for_submission rolls back if there is an error' do
+    track_progress @level_source.id
+
+    original_peer_reviews = PeerReview.where(level_source_id: @level_source.id)
+    PeerReview.stubs(:create!).raises(Exception, "Some error")
+
+    Honeybadger.stubs(:notify)
+
+    assert_raises(Exception) do
+      track_progress @level_source.id
+    end
+
+    assert original_peer_reviews == PeerReview.where(level_source_id: @level_source.id)
+  end
+
+  test 'notify honeybadger if new entries are not created' do
+    PeerReview.stubs(:create_for_submission)
+    Honeybadger.stubs(:notify)
+    Honeybadger.expects(:notify).never
```

Comment:
does this line mean we are not expecting it to get notified? that seems at odds with the test name